brand new story

```diff
@@ -0,0 +1 @@
+It was 5 am in the morning. John Doe was looking at his steaming cup of coffee and was thinking about the previous 72 hours of his life.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.